src/server/journalism.js

```diff
@@ -9,6 +11,49 @@ const client =
     });
 
 export const journalismTeam = (res, team) => {
+  return newPromiseChain()
+    .then(() => db.query('SELECT FROM team WHERE name = :name LIMIT 1', { params: { name: team } }))
+    .then((results) => {
+      const searchTeamDbInfo = results[0];
+      return { dbInfo: searchTeamDbInfo };
+    }).then((all) => {
+      return newPromiseChain()
+        .then(() => fetchFromFootballAPI(`http://api.football-data.org/v1/teams/${all.dbInfo.id}/fixtures`))
+        .then((data) => ({ ...all, footballApiData: data }));
+    }).then((all) => (
+      getTeamInformation(team).then((results) => ({ ...all, leftTeam: results }))
+    )).then((all) => {
+      return Promise.all(
+        all.footballApiData.fixtures.slice(0, 2).map(
```

Comment:
Whoops, this was for debugging.
I'll change it to 10 because otherwise it returns like 20 - 30 results - unless we want all of them?